<ui version="4.0" >
 <class>QgsAddAttrDialogBase</class>
 <widget class="QDialog" name="QgsAddAttrDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Attribute</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOkButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="mTypeBox" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Type:</string>
     </property>
     <property name="buddy" >
      <cstring>mTypeBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="mNameEdit" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Name:</string>
     </property>
     <property name="buddy" >
      <cstring>mNameEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>mNameEdit</tabstop>
  <tabstop>mTypeBox</tabstop>
  <tabstop>mOkButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
